packages/pythonlib/pythonlib.v0.12.0/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "opensource@janestreet.com"
+authors: ["Jane Street Group, LLC <opensource@janestreet.com>"]
+homepage: "https://github.com/janestreet/pythonlib"
+bug-reports: "https://github.com/janestreet/pythonlib/issues"
+dev-repo: "git+https://github.com/janestreet/pythonlib.git"
+license: "MIT"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml"      {>= "4.07.0"}
+  "base"       {>= "v0.12" & < "v0.13"}
+  "ppx_jane"   {>= "v0.12" & < "v0.13"}
+  "ppx_python" {>= "v0.12" & < "v0.13"}
+  "stdio"      {>= "v0.12" & < "v0.13"}
+  "typerep"    {>= "v0.12" & < "v0.13"}
+  "dune"       {build & >= "1.5.1"}
```

Comment:
```suggestion
  "dune"       {>= "1.5.1"}
```
Dune should not be flagged as a build dependency anymore (see #14266)

Reply:
OK, sorry for that.